src/customHooks/useDisruptionAffectedItems.js

```diff
@@ -47,20 +58,34 @@ const useDisruptionAffectedItems = (disruption) => {
       </div>
       <div className="wmnds-col-1">
         {/* Affected Services / Bus */}
-        {disruption.servicesAffected &&
-          disruption.mode === 'bus' &&
-          disruption.servicesAffected
-            .sort((a, b) => a.serviceNumber.replace(/\D/g, '') - b.serviceNumber.replace(/\D/g, ''))
-            .map((affected) => (
-              <FavBtn
-                id={affected.id}
-                severity={disruption.disruptionSeverity}
-                text={affected.serviceNumber}
-                title={`${affected.routeDescriptions[0].description} (${affected.operatorName})`}
-                mode={disruption.mode}
-                key={affected.id}
+        {disruption.servicesAffected && disruption.mode === 'bus' && (
+          <>
+            {disruption.servicesAffected
+              .sort(
+                (a, b) => a.serviceNumber.replace(/\D/g, '') - b.serviceNumber.replace(/\D/g, '')
```

Comment:
The sort needs changing slightly as i've searched for the 997:https://deploy-preview-429--wmn-disruptions.netlify.app/?isMapVisible=false&mode=bus&query=997&selectedItem=4738 but can't see it as it is "hidden" behind the button.

If a user searches for a service it should appear first so they can confirm the disruption is relevant to them.

Reply:
V good point